### ISSUE TITLE
use of `async` as argument name causing issues with swig (and grc on gr39)

### ISSUE THREAD

**Issue by jacobagilbert** [ghostop14/gr-xcorrelate#1]
The `xcorrelate` block takes `async` as an argument which caused swig to explode during build with this error in 3.8

```bash
    def make(num_inputs: 'int', signal_length: 'int', data_type: 'int', data_size: 'int', max_search_index: 'int', decim_frames: 'int', num_outputs: 'int', async: 'bool'=False, normalize: 'bool'=True) -> "gr::xcorrelate::xcorrelate::sptr":
                                                                                                                                                            ^
SyntaxError: invalid syntax
```

I tried building on 3.9 and it built as SWIG is gone, but GRC would not generate the flowgraph because async is in the yaml block definition. This also causes problems for 3.8.

```bash
gnuradio.grc.core.errors.TemplateError: (TemplateError('xcorrelate.xcorrelate(${num_inputs},${signal_length},${type.data_type},${type.size},${max_search_offset},${decim_frames},${zero},${async},${normalize})', "(SyntaxError) invalid syntax (<unknown>, line 1) ('async') at line: 1 char: 130"), 'xcorrelate.xcorrelate(${num_inputs},${signal_length},${type.data_type},${type.size},${max_search_offset},${decim_frames},${zero},${async},${normalize})')
```

This is with python 3.8.5 on Ubuntu 20.04 (GR 3.8.2) which appears to treat `async` as special.

```python
>>> async=1
  File "<stdin>", line 1
    async=1
         ^
SyntaxError: invalid syntax
````

Changing the argument name fixes things.

**Comment by ghostop14** [comment]
Hi Jacob, I haven't run into that and I've been running 3.8.2 on Ubuntu 20 as well.  But I did update the param name.  Pull again and see if that helps.  Also, the Ref Correlate FD version has much better performance than the TD version (typical convolution vs. multiplication), so you may want to look at that as well.

**Comment by jacobagilbert** [comment]
Tested and resolved on 3.8. I haven't got far with this module but I'll check the FD version out. 

I don't understand the nature of this error (or why it ever worked) but such is life. Thanks!
